docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -3921,6 +3921,25 @@ dataset. Select the guardian at random.
   or for all duplicated rows to appear at the end of the dataset, so we would
   need a strategy to address this.
 
+.. warning::
+
+  We `tried sorting <https://github.com/ihmeuw/pseudopeople/pull/384>`_
+  the decennial census by year and household ID (`SSCI-1677
+  <https://jira.ihme.washington.edu/browse/SSCI-1677>`_, `MIC-4833
+  <https://jira.ihme.washington.edu/browse/MIC-4833>`_) so that
+  simulants duplicated at a guardian's household would appear next to
+  other members of the guardian's household in the returned dataset.
+  However, the runtime of the sort was unacceptably slow for the USA
+  data, so we `reverted the change
+  <https://github.com/ihmeuw/pseudopeople/pull/386>`_. In the current
+  implementation, duplicates are appended to the end of the DataFrame,
+  but this happens at the *shard* level. Thus, while the sample data has
+  all duplicates appearing at the end of the dataset (making them easy
+  to find), duplicates in the USA or RI data appear at the end of each
+  shard, which is less obvious. We deemed this acceptable behavior for
+  the time being, though it may be worth revisiting the order of rows in
+  each dataset as we continue to make improvements.
```

Comment:
Do we have a ticket for the engineering part of this?  I think we should make sure that such a ticket exists, and link to it here.

Reply:
I don't think I can make an engineering ticket for this yet because we don't know what we want, but I can create a research ticket to circle back to this.

Reply:
> I don't think I can make an engineering ticket for this yet because we don't know what we want, but I can create a research ticket to circle back to this.

I thought we decided that we would have them sort by household id as part of the "post-processing" of the data produced by the simulation. But as long as we have a ticket somewhere to remember to revisit this it is fine.

Reply:
I think it was unclear that grouping by year and household ID was the best ordering -- for example, we may want to group by state as well. I made a ticket to revisit this ([SSCI-1794](https://jira.ihme.washington.edu/browse/SSCI-1794)) and added a link to it in the concept model.